README/RCodingStyle.cxx

```diff
@@ -0,0 +1,83 @@
+/// \file RCodingStyle.cxx
```

Comment:
Sometimes it makes sense to add `\brief` section which describes the contents and the purpose of the files.